src/relay/WalletSDKRelay.ts

```diff
@@ -1044,18 +1038,26 @@ export class WalletSDKRelay extends WalletSDKRelayAbstract {
           resolve(response as SwitchEthereumChainResponse);
         });
 
-        const _cancel = (errorCode?: number) => {
-          if (errorCode) {
+        const _cancel = (error?: any) => {
+          if (typeof error === "number") {
```

Comment:
this function used to take a param without explicit type definition.
![image](https://user-images.githubusercontent.com/966662/165152599-7321197b-26e5-49b6-b802-106638b86b5d.png)
for backward compatibility, `_cancel` now takes `any` type and check whether it's number from legacy env.

Reply:
nitpicking question, could we make the optional error type `WalletUIError | number` instead of `unknown`? This way we have backwards compatibility without opening this function up to accept whatever as an arg.